2-mandatory/1-writers.js

```diff
@@ -71,7 +79,19 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  writers.forEach((writer) => {
+    const firstName = writer.firstName
```

Comment:
Please follow the example above. You don't have to create another variable in the loop. The loop is already iterating through the array.